src/mpl/src/timer/mpl_timer_clock_gettime.c

```diff
@@ -10,6 +10,10 @@ MPL_SUPPRESS_OSX_HAS_NO_SYMBOLS_WARNING;
 
 #if MPL_TIMER_KIND == MPL_TIMER_KIND__CLOCK_GETTIME
 
+#include "mpl_timer_common.h"
+
+static double tickval;
+
```

Comment:
How about move this variable into `mpl_timer_common.c`, then declare `extern double tickval;` here?

Reply:
Yeah, that would work too. We should probably namespace it as `MPLI_tickval` or something to be more in line with the coding standards. I can make that change if you agree.

Reply:
The idea is to keep `tickval` as private within `mpl`, right? A namespace would imply a public interface.

Reply:
I'm using `MPLI_` to convey internal namespace since these are defined in a header file. Without namespace, these have the (unlikely) potential to collide at the file level. Obviously we've survived without it for now, just trying to be consistent with other parts of MPICH.